### PR TITLE
[Feature] Implement call denied behaviour to return to setup view

### PR DESCRIPTION
**Pull request by jimchou-dev**
## Purpose

* Implement call denied behaviour to return to setup view

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
Verify that the following are valid
* When being denied from a Teams call will return to setupView similar to call eviction behaviour

## Other Information

* Added localization keys for snackbar for TD translation service

### CODE REVIEW

**Review by JoshuaLai — APPROVED** [review]
Tested and working as expected. 
Just one nit pick comment, I did see that you included a few new localized keys for things not applicable to this PR. Not sure if that was intended

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -34,6 +34,10 @@
 "AzureCommunicationUICalling.LobbyView.Text.WaitingForHost"="Waiting for host"; // host (people not server) of a meeting
 "AzureCommunicationUICalling.LobbyView.Text.WaitingDetails"="Someone in the meeting will let you in soon";
 
+/* OnHoldView */
+"AzureCommunicationUICalling.OnHoldView.Text.OnHold"="You're on hold";
+"AzureCommunicationUICalling.OnHoldView.Button.Resume"="Resume";
+
```
Comment (JoshuaLai) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings:40]: This doesn't look applicable to this PR. Was this intentional.

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -89,3 +94,6 @@
 "AzureCommunicationUICalling.SnackBar.Text.ErrorCallEnd"="You were removed from the call due to an error.";
 "AzureCommunicationUICalling.SnackBar.Text.ErrorCallEvicted"="You were removed from the call.";
 "AzureCommunicationUICalling.SnackBar.Text.Error"="There was an error.";
+"AzureCommunicationUICalling.SnackBar.Text.ErrorResumeCallTitle"="Unable to resume call";
+"AzureCommunicationUICalling.SnackBar.Text.ErrorResumeCallSubTitle"="Your mic is in use by another app";
```
Comment (JoshuaLai) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings:98]: This doesn't look applicable to this PR. Was this intentional.

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -43,6 +47,7 @@
 "AzureCommunicationUICalling.CallingView.ParticipantDrawer.LocalParticipant"="%@ (me)"; // %@ is local participant name
 "AzureCommunicationUICalling.CallingView.ParticipantDrawer.IsMuted"="Muted";
 "AzureCommunicationUICalling.CallingView.ParticipantDrawer.IsUnmuted"="Unmuted";
+"AzureCommunicationUICalling.CallingView.ParticipantDrawer.OnHold"="On hold";
```
Comment (JoshuaLai) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings:50]: This doesn't look applicable to this PR. Was this intentional.

**Review by kevinyulu — APPROVED** [review]
tested and LGTM

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings (hunk 3 above)
Comment (jimchou-dev) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings:50]: Yes its intentional to be send to TD service for translation all in one file

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (jimchou-dev) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings:98]: Yes its intentional to be send to TD service for translation all in one file

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (jimchou-dev) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Localization/en.lproj/Localizable.strings:40]: Yes its intentional to be send to TD service for translation all in one file

**Pull request merged**